CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* PayPalNativeCheckout
+  * Bumping native-checkout version to 0.8.2
+  * Bugfix to support logging out of session when client ID changes
```

Comment:
Is this the same bugfix as mentioned on iOS [here](https://github.com/braintree/braintree_ios/pull/872/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed)?

Reply:
yup thats corect, I'll match up the text with theirs